packages/web-api/src/response/AdminAnalyticsGetFileResponse.ts

```diff
@@ -20,3 +21,73 @@ export type AdminAnalyticsGetFileResponse = WebAPICallResult & {
 export interface ResponseMetadata {
   messages?: string[];
 }
+
+export interface MemberDetails {
```

Comment:
What do you think naming like `AdminAnalyticsMemberDetails` or so? `MemberDetails` cannot be exposed to external as it's too general.